HISTORY.rst

```diff
@@ -49,6 +81,13 @@ unreleased
 
 * Data History commands, under the `az dt data-history` command group, are now GA.
 
+**IoT Central updates**
+
+* Added support for simulating device identities that use x509 thumbprint or CA authentication, impacting the
```

Comment:
These updates go under the IoT device updates section, you could basically undo the existing removal

![image](https://user-images.githubusercontent.com/28658112/177883072-39b156d4-7c5e-4287-8bd8-01b248b05a97.png)